idseq_dag/util/log.py

```diff
@@ -131,7 +151,7 @@ def some_method
     {"event": "ctx_start", "context_name": "sub_step", "caller": {"filename": "some_file.py", "method": "some_method"}, "values": {abc": 123}}
     {"event": "ctx_end", "context_name": "sub_step", "caller": {"filename": "some_file.py", "method": "some_method"}, "values": {abc": 123}, "duration_ms": 5006} (5.0 seconds)
     '''
-    extra_fields = {"context_name": context_name}
+    extra_fields = {"context_name": context_name, "uid": secrets.token_hex(6)}
```

Comment:
what will this uid field be used for?

Reply:
This is useful to correlate start and end events, specially when we are trying to detect hanging issues. If you don't find an "ctx_end" counterpart with the same uid, that means the step has never completed.